src/ddmd/cppmangle.d

```diff
@@ -563,7 +563,7 @@ public:
         }
         else
         {
-            t.error(Loc(), "Internal Compiler Error: unsupported type %s\n", t.toChars());
+            t.error(Loc(), "Internal Compiler Error: type %s can not be mapped to C++\n", t.toChars());
```

Comment:
Can you change ` %s ` to `` `%s` ``, so the error message could get syntax highlighting?